models/repo.go

```diff
@@ -2043,7 +2054,7 @@ func DeleteRepository(doer *User, uid, repoID int64) error {
 	}
 
 	attachmentPaths := make([]string, 0, 20)
-	attachments := make([]*Attachment, 0, len(attachmentPaths))
+	attachments = make([]*Attachment, 0, len(attachmentPaths))
```

Comment:
Why do we need to get the attachment list again?

Reply:
The `attachment` table stored issue/comment attachments and release attachments. Second is get issue/comment attachments list.